mysite/announcements/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+from .models import Announcements
+from django.contrib.auth.models import User, Group
+
+
+class AnnouncementsAdmin(admin.ModelAdmin):
+    list_display = ('title', 'message', 'date_time_expire')
+    change_form_template = "announcements/announcements_change_form.html"
+
+    def response_change(self, request, obj):
+        if "_send_announcement" in request.POST:
+            group_list = request.POST.getlist('groups')
+            group_users = User.objects.filter(groups__id__in=group_list).distinct()
```

Comment:
Why are we again fetching all columns of user here? Can we just fetch `id`\`username`?

Reply:
I'll check this out.
Select * from user where id in grouplist

Reply:
@nishanty13 
1. Check out what?
2. SQL query is wrong.